packages/fluentui/docs/src/examples/components/RadioGroup/Variants/RadioGroupCustomVerticalExample.shorthand.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Divider, RadioGroup, Input } from '@fluentui/react-northstar';
+
+class RadioGroupCustomVerticalExample extends React.Component {
+  state = { selectedValue: '', inputTabIndex: '-1' };
+
+  render() {
+    const { selectedValue } = this.state;
+    return (
+      <div style={{ maxWidth: '400px' }}>
+        The selected value is: {selectedValue}
+        <Divider />
+        <RadioGroup
+          vertical
```

Comment:
A single difference between these examples is `vertical` prop?

Reply:
Yep, same with "basic" examples for RadioGroup

Reply:
Simplified